includes/widgets/button.php

```diff
@@ -314,16 +314,20 @@ protected function _register_controls() {
 			]
 		);
 
-		$this->add_control(
-			'background_color',
+		$this->add_group_control(
+			Group_Control_Background::get_type(),
 			[
-				'label' => __( 'Background Color', 'elementor' ),
-				'type' => Controls_Manager::COLOR,
-				'global' => [
-					'default' => Global_Colors::COLOR_ACCENT,
-				],
-				'selectors' => [
-					'{{WRAPPER}} .elementor-button' => 'background-color: {{VALUE}};',
+				'name' => 'background',
```

Comment:
Did it keep backward compatibility?
Try to save a color with the old control, and then change it to the group control, did the old color still exist?

Reply:
Now it will